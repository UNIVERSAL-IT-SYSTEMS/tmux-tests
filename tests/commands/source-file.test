#!/bin/sh

F=$(mktemp)
trap 'rm -f $F' 0 1 15

cat <<EOF >$F
un\
bind \
-a
EOF
$TMUX_L -f $F start\; lsk
$TMUX_L start\; source $F\; lsk

cat <<EOF >$F
   unbind -a   
EOF
$TMUX_L -f $F start\; lsk
$TMUX_L start\; source $F\; lsk

cat <<EOF >$F
	unbind -a
EOF
$TMUX_L -f $F start\; lsk
$TMUX_L start\; source $F\; lsk

cat <<EOF >$F
run "$TMUX_L unbind -a"
EOF
$TMUX_L -f $F start\; lsk
$TMUX_L start\; source $F\; lsk

unset FOO
cat <<EOF >$F
FOO=123
EOF
$TMUX_L start\; source $F\; showenv -g FOO

cat <<EOF >$F
	unbind \ 
	-a
EOF
$TMUX_L start\; source $F 2>&1|sed "s|$F|@|"

cat <<EOF >$F
burble
EOF
$TMUX_L start\; source $F 2>&1|sed "s|$F|@|"
