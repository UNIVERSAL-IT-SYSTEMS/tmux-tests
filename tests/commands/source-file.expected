FOO=123
@:2: unknown command: -a
@:1: unknown command: burble
